app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -20,45 +20,46 @@ def hash_for_pdf
       spouse_ssn = get_spouse_ssn
       answers = {
         # county code
-        "CM4": county_code[1],
-        "CM3": county_code[2],
-        "CM2": county_code[3],
-        "CM1": county_code[4],
+        CM4: county_code[1],
+        CM3: county_code[2],
+        CM2: county_code[3],
+        CM1: county_code[4],
 
         # taxpayer ssn
-        "undefined": taxpayer_ssn[0],
-        "undefined_2": taxpayer_ssn[1],
-        "Your Social Security Number required": taxpayer_ssn[2],
-        "Text3": taxpayer_ssn[3],
-        "Text4": taxpayer_ssn[4],
-        "Text5": taxpayer_ssn[5],
-        "Text6": taxpayer_ssn[6],
-        "Text7": taxpayer_ssn[7],
-        "Text8": taxpayer_ssn[8],
+        undefined: taxpayer_ssn[0],
+        undefined_2: taxpayer_ssn[1],
+        'Your Social Security Number required': taxpayer_ssn[2],
```

Comment:
[dust] I thought CFA said they prefer double-quoted strings

Reply:
Huh, this was a rubocop autofix thing, and when I tried to change it to double quotes I got `Style/QuotedSymbols: Prefer single-quoted symbols when you don't need string interpolation or special symbols.`

I wonder if the default rubocop config is not aligned with CfA preferred style and there's no override for this one in the local config?